### PR TITLE
Fixes #496 - Allow skipping lines when reading xls/xlsx/csv files

### LINKED ISSUE #496: Allow to skip first nth lines when reading a file
It would be nice to be able to skip the first lines when reading a file. The use case is especially frequent with Excel files, and when you want to read files without editing them first. Patch follows.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: docs/formats.rst [hunk 1]
```diff
@@ -40,6 +40,13 @@ is headers with the ``headers`` boolean parameter (defaults to ``True``)::
 
     tablib.import_set(your_data_stream, format='csv', headers=False)
 
+It is also possible to provide the ``skip_lines`` parameter to a number of
```
Comment (hugovk) [inline docs/formats.rst:43]:
```suggestion
It is also possible to provide the ``skip_lines`` parameter for the number of
```

File: src/tablib/formats/_csv.py [hunk 2]
```diff
@@ -42,8 +42,9 @@ def import_set(cls, dset, in_stream, headers=True, **kwargs):
 
         rows = csv.reader(in_stream, **kwargs)
         for i, row in enumerate(rows):
-
-            if (i == 0) and (headers):
+            if i < skip_lines:
+                continue
+            if (i == skip_lines) and (headers):
```
Comment (hugovk) [inline src/tablib/formats/_csv.py:47]: Remove redundant parentheses, like in `src/tablib/formats/_xls.py`:

```suggestion
            if i == skip_lines and headers:
```

File: src/tablib/formats/_xlsx.py [hunk 3]
```diff
@@ -85,8 +85,10 @@ def import_set(cls, dset, in_stream, headers=True, read_only=True):
         dset.title = sheet.title
 
         for i, row in enumerate(sheet.rows):
+            if i < skip_lines:
+                continue
             row_vals = [c.value for c in row]
-            if (i == 0) and (headers):
+            if (i == skip_lines) and (headers):
```
Comment (hugovk) [inline src/tablib/formats/_xlsx.py:91]:
```suggestion
            if i == skip_lines and headers:
```

**Review by hugovk — APPROVED** [review]
Few minor suggestions, looks good.

**Comment by claudep** [comment]
Thanks for the review!

**Pull request merged**